lib/msun/src/e_pow.c

```diff
@@ -299,7 +299,9 @@ pow(double x, double y)
 	r  = (z*t1)/(t1-two)-(w+z*w);
 	z  = one-(r-z);
 	GET_HIGH_WORD(j,z);
-	j += (n<<20);
+    /* sign bit of z is 0.
+       sign bit of j will indicate sign of 0x3ff-biased exponent */
+	j += (int32_t)((u_int32_t)n<<20);
```

Comment:
We prefer uint32_t instead of u_int32_t generally in the FreeBSD tree, especially for new code.

Reply:
I can change that too.
`u_int32_t` is used elsewhere in this file and in most of the occurrences in this directory.
`uint32_t` is used in a few files. Both `u_int32_t` and `uint32_t` are used in only two.